### DISCUSSION TITLE
Suggestion - Auto Expand Matrix View

### DISCUSSION THREAD

**Discussion by unfor19** [community/community#11275] [Actions] [answered]
I'm using GitHub Actions [matrix](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) builds, this is how it looks like today:

<img width="1771" alt="Screen Shot 2022-02-10 at 1 44 37" src="https://user-images.githubusercontent.com/15122452/153316521-48e1b7e7-3ca1-448f-8dd1-dc6c9c6117d1.png">

The above image doesn't provide much info and it would be great if there was a checkbox for setting "auto-expand matrix". The expected result:

<img width="1780" alt="Screen Shot 2022-02-10 at 1 44 54" src="https://user-images.githubusercontent.com/15122452/153316537-70d5dc25-4a68-4efa-a30a-37d5dbd79f6e.png">

Yup, I manually clicked on each matrix to see more details.

I think phase 2 can be - "auto-expand matrix if less than X jobs", but let's start with a simple boolean checkbox ;)

If there's any way I can contribute to the code, let me know how and I'll be glad to do it myself.

**Comment by ethomson** [comment, marked as answer]
Thanks for the suggestion @unfor19, we'll add this to the backlog!

**Comment by alshdavid** [comment]
Any progress here?

**Comment by ReenuBairi** [comment]
Hello @ethomson any updates on this backlog?

**Comment by oliversaggau** [comment]
Any progress here?
